scripts/tests/yaml/runner.py

```diff
@@ -256,7 +256,10 @@ def runner_base(ctx, configuration_directory: str, test_name: str, configuration
     tests_finder = TestsFinder(configuration_directory, configuration_name)
 
     parser_config = TestParserConfig(pics, specifications, kwargs)
-    parser_builder_config = TestParserBuilderConfig(tests_finder.get(test_name), parser_config, hooks=TestParserLogger())
+    test_list = tests_finder.get(test_name)
+    if len(test_list) == 0:
+        raise Exception(f"No tests found for test name '{test_name}'")
+    parser_builder_config = TestParserBuilderConfig(test_list, parser_config, hooks=TestParserLogger())
```

Comment:
Maybe this code block can be moved right after `tests_finder = TestsFinder(configuration_directory, configuration_name)` ?

Reply:
Done.